### DISCUSSION TITLE
Interpolation grid in v2

### DISCUSSION THREAD

**Discussion by nedam321** [reaktoro/reaktoro#342] [Q&A]
### Description

What is the equivalent of setTemperatures and setPressures - member functions of ChemicalEditor class in Reaktoro v2?

### What is your operating system?

None

**Comment by allanleal** [comment]
There is no interpolation grid in Reaktoro v2 for the standard thermodynamic properties. In Reaktoro v2, these properties are fully computed by the underlying standard thermodynamic model of the species (e.g. HKF for aqueous species). These models are optimized with memoization. For repeated evaluations on the same (T, P) conditions, the cached properties are returned.
